recipe/build.sh

```diff
@@ -1,34 +1,17 @@
-cd src/cxx
-
-autoconf
-
-export CPATH="${PREFIX}/include"
 
-./configure --prefix=$PREFIX --enable-noisy-make
+cd src/common_libraries/libsharp
 
-# We need to manually add -fPIC because there is no way
-# to force healpix to use it otherwise. We accomplish this
-# by changing the status and then instantiating it
+./configure --prefix=$PREFIX --disable-silent-rules --disable-dependency-tracking
 
-if [ "$(uname)" == "Darwin" ]; then
-    
-    sed -i '' 's/-O2/-O2 -fPIC/g' config.status
-    sed -i '' 's/-O3/-O3 -fPIC/g' config.status
+make install -j ${CPU_COUNT}
 
-else
+cd -
 
-    sed -i 's/-O2/-O2 -fPIC/g' config.status
-    sed -i 's/-O3/-O3 -fPIC/g' config.status
-    
-fi
```

Comment:
Are you sure we don't still need -fpic?

Reply:
It makes sense to have `-fPIC`, I will put it back in. This was one of the inherited changes from the `3.70` pull request. I have added the `--enable-shared`.

Reply:
Why did you add enabled shared? Vendoring libsharp shared libs will make this package conflict with the existing libsharp packages.

Reply:
Ahh okay, that make sense a I have removed the `--enable-shared` from the libsharp configure and instead added the `-fPIC` flag based on the way it was done before.